pkg/apiutil/errors.go

```diff
@@ -188,4 +188,10 @@ var (
 
 	// ErrMissingDomainID indicates missing domainID.
 	ErrMissingDomainID = errors.New("missing domainID")
+
+	// ErrMissingUserName indicates missing user name.
+	ErrMissingUserName = errors.New("missing user name")
+
+	// ErrMissingProfilePicture indicates missing profile picture.
+	ErrMissingProfilePicture = errors.New("missing profile picture")
```

Comment:
This error is not used. How comes that CI linter does not report it? Please check.